src/sql/workbench/browser/modal/modal.ts

```diff
@@ -589,6 +622,15 @@ export abstract class Modal extends Disposable implements IThemable {
 			this._modalDialog.style.borderWidth = border ? '1px' : '';
 			this._modalDialog.style.borderStyle = border ? 'solid' : '';
 			this._modalDialog.style.borderColor = border;
+
+			calloutStyle.setProperty('--border', `${border}`);
```

Comment:
How are these different than using the properties directly on the style object?

Reply:
I am using these values for CSS pseudo element properties which I cannot add in the context here.
modal.css, lines 27-57 are multiple instances of these values being passed in.